processing/src/main/java/org/apache/carbondata/processing/util/CarbonDataProcessorUtil.java

```diff
@@ -604,4 +606,33 @@ public static boolean isHeaderValid(String tableName, String header,
     }
     return dateformatsHashMap;
   }
+
```

Comment:
Please remove this reflection code and try to set from RDD like how it was doing in CarbonDataLoadRDD